transmartApp/grails-app/controllers/org/transmartproject/interceptors/ConceptsInterceptor.groovy

```diff
@@ -1,12 +1,11 @@
 package org.transmartproject.interceptors
 
-import org.transmart.audit.AuditLogService
 import org.transmartproject.core.users.User
 
 
 class ConceptsInterceptor {
 
-    AuditLogService auditLogService
+    def auditLogService
```

Comment:
This makes it do by-name bean insertion, by type is more robust. Use an @Autowired if needed. You can also update the studyIdService bean in the next line while you're at it.